jenkins/shared-library/vars/createMultibranchJob.groovy

```diff
@@ -78,20 +94,15 @@ void call(Map jobConfiguration) {
 											parentCredentials(true)
 										}
 									}
+									gitHubStatusChecks {
```

Comment:
what does this do?

Reply:
Sorry, I forgot to mention this.
By default, all the GitHub check status gets reported under ``Jenkins``.  This causes issues with GitHub since the status check will overwrite each other since Jenkins does not combine them.
This will allow each job to report their status to GitHub under a different name.  You will see a lot more information in GitHub. 😄